sink_service/source/main.c

```diff
@@ -82,7 +144,7 @@ int main(int argc, char * argv[])
                 break;
             case 'p':
                 /* Get the port name */
-                port_name = optarg;
+                strncpy(port_name, optarg, MAX_SIZE_PARAMETER - 1);
```

Comment:
And remove this copy (and MAX_SIZE_PARAMETER const)

Reply:
Ok, I forgot I had added this one as well. One problem I see now is that we can end up with excessive filenames, at least the tests are now failing due to too big strings.